File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/base/fragment/binding/Binding.java
package ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding;

import android.view.View;

/**
 * @author meugen
 */
public interface Binding {

    void attachView(View view);

    void detachView();

    <V extends View> V get(int id);

    boolean has(int id);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/base/fragment/binding/Binding.kt
package ua.meugen.android.pdfcomposer.ui.activities.base.fragment.binding

import android.view.View

/**
 * @author meugen
 */
interface Binding {

    fun attachView(view: View)

    fun detachView()

    operator fun <V : View> get(id: Int): V

    fun has(id: Int): Boolean
}
